examples/relationships/README.md

```diff
@@ -0,0 +1,209 @@
+
+Schema extension are being performed right in the web site source code (in [gatsby-node.js](./gatsby-node.js) file) using [`createSchemaCustomization`](https://www.gatsbyjs.com/docs/node-apis/#createSchemaCustomization) method.
+
+If you want to separate this logic, it is possible to create a [local Gatsby plugin](https://www.gatsbyjs.com/docs/creating-a-local-plugin/) and define these customization there, as it is showcased in the [Kontent Lumen starter](https://github.com/Kentico/gatsby-starter-kontent-lumen/tree/master/plugins/kontent-used-by-content-items), just use [`createResolvers`](https://www.gatsbyjs.com/docs/node-apis/#createResolvers) method for the customizations. If you want to re-use this code, you could [publish the plugin to the npm](https://www.gatsbyjs.com/contributing/submit-to-plugin-library/) and then re-use it.
+
+> In case you published your schema extension bound to Kontent GraphQL nodes feel free to raise an issue/pull request with plugin description and we are happy to mentions it here!
+
+### Language variant relationships
+
+The example is describing possibility of creating relationships among language variants. This relationship could be ensured by [extending schema definition](https://www.gatsbyjs.com/docs/schema-customization).
+
+> If you wish to use the Gatsby for bigger multilingual, take a look to the [blog post "How to Build Multilingual Sites with Gatsby"](https://www.gatsbyjs.com/blog/2020-02-19-how-to-build-multilingual-sites-with-gatsby)
+
+Actual use case is to extend `Article` by two fields: `fallback_used` and `other_languages`.
+
+- `fallback_used` is a `Boolean` field saying it the specified language variant [used language fallback](https://docs.kontent.ai/tutorials/manage-kontent/projects/set-up-languages#a-language-fallbacks) or not.
+- `other_languages` is a `[kontent_item_article]` type, which means it is an array of other GraphQL nodes with type `kontent_item_article`, which is the type representing `Article` (you could find type name in `internal.type` field or you could use `getKontentItemNodeTypeName` method from Kontent Gatsby source plugin if you know content type codename).
+
+#### Language fallback extension showcase
+
+For the real-life example, there is a listing ot the articles registered on the `/articles` route. This page is registered in `gatsby-node` file using [`createPages`](https://www.gatsbyjs.com/docs/node-apis/#createPages) API method to register the page using [articles.js](./src/templates/articles.js) template. This template is using `fallback_used` element to filter out the listing.
+
+So in english (en-US) route `/en-US/articles`, you will see all articles:
+
+![English article listing](./docs/en-articles-listing.png)
+
+But in czech version (cs-CZ), there is only one article listed because the only article is translated into Czech language.
```

Comment:
```suggestion
But in the Czech version (cs-CZ), there is only one article listed because the only article is translated into the Czech language.
```